### PR TITLE
messaging

### PR DESCRIPTION
**Pull request by wolffles**
description: added a method to check and send messages
test coverage: 0%

sorry I sent so many messages, so I found out that response is actually just a blank string when messages are sent, so I thought maybe we don't need to parse the object cuz it's just blank

### CODE REVIEW

File: lib/bapig.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ def initialize(email, password)
     puts response.code
     raise "invalid email/password " if response.code == 401
     @auth_token = response["auth_token"]
+    @auth_user = response["user"]
```
Comment (brittanyjspringer) [inline lib/bapig.rb:14]: I'm not clear on what the point of this variable is. It is not harming anything but I want you to be mindful of creating class variables if you do not use them.

Comment (brittanyjspringer) [reply lib/bapig.rb:14]: @wolffles ^

**Comment by brittanyjspringer** [comment]
OK, good! I gotta a little lost in the messages so I decided to start here. Glad I did! Please submit your checkpoints with a link to your PRs, please. Thanks!

File: lib/bapig.rb (hunk 1 above)
Comment (wolffles) [reply lib/bapig.rb:14]: oh yeah I was trying to create a variable to get emails of the user but i had some trouble working with it, and i went the easier route and just forgot to remove it

**Pull request merged**